src/api/index.js

```diff
@@ -16,11 +14,17 @@ export async function query(graphqlQuery, variables) {
   const json = await response.json()
   return json
 }
-
-export async function tasks(variables) {
-  return query(Tasks, variables)
-}
-
-export async function createTask(variables) {
-  return query(CreateTask, variables)
+export async function mutate(graphqlQuery, variables) {
```

Comment:
Is this required? 🤔 This looks as same as `query`

Reply:
for now, I will generate uuid in it later this pr.

Reply:
🙆‍♂️